source/apps/copy.txt

```diff
@@ -0,0 +1,200 @@
+.. _copy-app:
```

Comment:
is this just move of `configure/copy-app.txt`?

Reply:
Not just a move, I also updated the procedures